modules/desktop/graphics/labwc.config.nix

```diff
@@ -298,6 +298,12 @@ in
         path = "${pkgs.labwc}/bin/labwc --exit";
         icon = "${pkgs.icon-pack}/system-log-out.svg";
       }
+      {
+        name = "Notepad Next";
+        description = "Notepad Application";
+        path = "${pkgs.notepad-next}/bin/NotepadNext";
```

Comment:
Not supported on `Aarch64` https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/editors/notepad-next/default.nix#L41
And run-checks are failing, please fix that.

Reply:
It is moved from launchers of labwc config to desktop service.